packages/app/src/client/views/ring-health/RingTable.tsx

```diff
@@ -113,9 +136,12 @@ const RingTable = ({ ringEndpoint, baseUrl }: Props) => {
               <TableCell>Zone</TableCell>
               <TableCell>Address</TableCell>
               <TableCell>Tokens</TableCell>
+              <TableCell>
+                <Typography variant="srOnly">Forget Shard</Typography>
+              </TableCell>
             </TableRow>
           </TableHead>
-          {shards ? (
+          {shards && !isRefreshing ? (
```

Comment:
Let's not show the loading screen on subsequent refreshes

Reply:
it won't. This just triggers the loading screen when clicking "Forget" (See the gif above) until the request is finished and the new shards are returned. 

But I see how this is not clear in the code.

Reply:
Ahh ok thanks for clearing that up

Reply:
So to summarise: 
The loading screen is only shown on initial load and when triggering the "forget" action. 

We can change that, but I'd believe we should show the user some sort of spinner after clicking "forget".
An option would be to just show a [circle spinner](https://material-ui.com/components/progress/#circular-indeterminate) where the button is. 
In effect that would be the same as using the [LoadingButton](https://next.material-ui.com/components/buttons/#loading-buttons), though the LoadingButton component requires an upgrade to material-ui 5.

Reply:
Let's save these ideas for some future improvements :-) We may actually want to centralize the state into redux at some point under our /state folder so we can surface up notifications when shards aren't ACTIVE. This is later though, lets see how what we have performs for users :-)